### ISSUE TITLE
Make point markers more circular

### ISSUE THREAD

**Issue by quincylvania** [openstreetmap/iD#6163]
Some of the boxier icons look crunched for space on the current point markers.

<img width="152" alt="Screen Shot 2019-04-08 at 3 03 28 PM" src="https://user-images.githubusercontent.com/2046746/55759956-9efe8380-5a0f-11e9-84c3-5dbe38256467.png">

We should make them more circular but leave a tiny pin part to indicate the exact location of the feature.

<img width="482" alt="Screen Shot 2019-04-08 at 3 00 43 PM" src="https://user-images.githubusercontent.com/2046746/55760448-f7825080-5a10-11e9-855f-bce417c87455.png">

This would also be more consistent with current trends in POI marking. 

<img width="361" alt="Screen Shot 2019-04-08 at 3 20 22 PM" src="https://user-images.githubusercontent.com/2046746/55760757-e259f180-5a11-11e9-949d-c4961af037c6.png">
<img width="188" alt="Screen Shot 2019-04-08 at 3 06 13 PM" src="https://user-images.githubusercontent.com/2046746/55760758-e2f28800-5a11-11e9-98b5-0521873ac0d4.png">

**Comment by bhousel** [comment]
Can we just make them circles? I don’t think we need a pin point.

**Comment by quincylvania** [comment]
> Can we just make them circles?

_Maybe_. One issue is that they would look identical to vertices.

The main issue with circles though is that that lat/lon of the feature is the center of the circle, meaning the marker obscures the most relevant area of the imagery. This isn't a big deal for large features like parks and business, but for tiny garbage bins and road signs, many users will want to place points more exactly. With pins the lat/lon is less obscured, which I think is better for an editing app. Apple Maps even uses pins when you add markers via the MapKit API:

<img width="137" alt="Screen Shot 2019-04-08 at 4 29 30 PM" src="https://user-images.githubusercontent.com/2046746/55763382-87c59300-5a1b-11e9-8b40-cd3a7b3ee57b.png">


<img width="255" alt="Screen Shot 2019-04-08 at 4 19 00 PM" src="https://user-images.githubusercontent.com/2046746/55763204-db83ac80-5a1a-11e9-9d83-88aedbdd151e.png">

**Comment by BjornRasmussen** [comment]
> Some of the boxier icons look crunched for space on the current point markers.

This does not seem like an issue to me, but if everyone else thinks it is, another solution would be to just display the image slightly smaller inside of the place icons.  This would fix the issue of a couple icons looking crunched.  

I really like how smooth the current design is, and also how the main icon is farther away from the point of the POI.
  
Another good way to keep the existing shape while solving the issue you mentioned would be creating a more beautiful version of this:
![image](https://user-images.githubusercontent.com/37753777/55765864-c5d4ae00-5a3f-11e9-93c6-3e8ad84b8ce0.png)
... basically just making the existing icon wider at the bottom to accommodate square icons.

**Comment by tordans** [comment]
My two cents:
- I don't feel the design pain in this case; IMO the current design is good (of not perfect)
- I like the proposed design; it solves the given issue; however, it is a bit harder to see the prick/pierce point (the old design is better in this sense)
- Whatever the design, IMO it needs to be a pin (not a circle), so there is no question about the lat/lng and also the pin does not obscure/overlap the prick/pierce-point.

PS: Thanks for making this easy to "consume", Quincy, with the preview images and such. Also for looking at those "small" design details <3.

**Comment by quincylvania** [comment]
I finally did this. I made the stem slightly longer than in the prior design so the head doesn't obscure the location so much. A big benefit I see is that the icons are now slightly larger and thus easier to discern.

<img width="1191" alt="Screen Shot 2019-12-09 at 5 22 46 PM" src="https://user-images.githubusercontent.com/2046746/70478490-26255f00-1aa9-11ea-9833-7c6c71dac4f1.png">
<img width="1280" alt="Screen Shot 2019-12-09 at 5 26 34 PM" src="https://user-images.githubusercontent.com/2046746/70478491-26255f00-1aa9-11ea-98d7-acae2748c537.png">

**Comment by BjornRasmussen** [comment]
Ok, not to be critical or anything, but I _really_ don't like this new shape.  
Here's the new design created by @quincylvania for reference:

![image](https://user-images.githubusercontent.com/37753777/70494371-1160bf80-1ad9-11ea-84e9-83483855c1ae.png)

The main problem is that it ends up looking really squashed on the bottom, and not visually appealing.  Due to an illusion, the perfect circle looks like some kind of squashed, upside-down pumpkin.

Here's a good example of what I'm talking about:

![image](https://user-images.githubusercontent.com/37753777/70494515-8c29da80-1ad9-11ea-8db8-07beb89cb361.png)

The Google logo isn't circular, but rather **intentionally misshapen to _look_ more circular**, and is, as a result, more aesthetically pleasing:

![image](https://user-images.githubusercontent.com/37753777/70494560-bc717900-1ad9-11ea-83e7-5feeb551b0a1.png)
 
So, what I think the new design you created it missing is some sort of transition between the icon and the pin that reduces the irritating squashed look.

Here is what Google Maps uses: 
![image](https://user-images.githubusercontent.com/37753777/70494672-3144b300-1ada-11ea-892f-08240bcdccb7.png)
Its pinhead is not _actually_ circular at the bottom, but by adding a more gradual transition from icon to point, making the icon less circular, they managed to make the pinhead _look_ more circular.  iD should employ a similar technique to ensure that the pinhead looks as good as possible.

In sum, I _do_ think that having a more circular icon is a good idea, but would prefer it to _look_ circular rather than actually _be_ circular if that makes sense.

**Comment by sun-geo** [comment]
A happy medium of these both shapes 
![image](https://user-images.githubusercontent.com/47290585/70539640-46feba80-1b64-11ea-8743-3962edb47623.png)
would be nice :-)

**Comment by quincylvania** [comment]
@BjornRasmussen @sun-geo Yeah, you're right the shape could still be improved. I don't really want it to look just like the Google or Apple pins but we'll see.
